### DISCUSSION TITLE
DualR2 - add ability to link relays so they present as a single unit, for high loads.

### DISCUSSION THREAD

**Discussion by faceless2** [arendst/Tasmota#15666] [Ideas]
A minor suggestion. I have a Sonoff Dual R2 and would like to link the two relays - I'm using it to switch a single 10A load (an EV charger) and sharing that over the two 10A relays makes sense.

I tried this by adjusting the template so "Relay 1" was assigned to two pins:

![image](https://user-images.githubusercontent.com/989243/169646822-8a2aabbf-e589-4e6c-b9df-db75f19ae783.png)

this was accepted and the UI presented the unit as a single relay, but only one of the two was actually switched.

My suggestion is simply: cater for a single relay being assigned more than one pin. It doesn't look like the UI needs any changes for this, so hopefully it would be a relatively simple change.

**Comment by faceless2** [comment]
Of course it was only immediately **after** posting this I found the concept of "Interlock" in the docs, which looks to be what I want. If this were an issue I'd close it, but not sure how with a Discussion.

**Reply by barbudor** [reply]
No need to close a discussion. Beauty of it😉

However you should soon find that Interlock is NOT what you are looking but the totally opposite. Interlock is meant to guarantee that 2 relay are never ON together. Like for driving motors in opposite directions.

In order to switch all relays of an R2 I would suggest to use POWER0 which does exactly that.

This being safe I'm very confident that this setup is safe. 2 physical relays will always have slight delay which, with high currents involved, could still be enough to spark.
For a safe high current setup I would use the tasmotized device to drive a power relay such as the ones intended for boilers.

**Reply by faceless2** [reply]
Thanks - yes, after trying interlock I realised that "power0" was actually what I was after. Works very nicely, thank you.

**Reply by sfromis** [reply]
Note that if safety or reliability depends on the load being distributed over both relays, and if the relays do not click at absolutely exactly the same time (due to software, electrical or mechanical differences), there is a risk that one relay briefly gets the whole load, and many types of load will have an extra spike of power draw when powered up. As a result of this, the improvement of using two relays may be quite marginal...

**Reply by faceless2** [reply]
> if safety or reliability depends on the load being distributed over both relays

Yes, quite agree, but not the case here - I load tested with one on it's own and its fine. It's mostly for longevity and because I have the copper and it seems a shame to waste it. Anyway it's outside and has nothing above it if it catches fire :-)
